README.md

```diff
@@ -1 +1,42 @@
-# plugin-Slack
\ No newline at end of file
+# Matomo Slack plugin
+
+Deliver the right information to stakeholders automatically, exactly when they need it. Save time with scheduled delivery to Slack channels, email, or mobile instead of manual sharing. Try now - it’s free!
+
+[![Build Status](https://github.com/matomo-org/plugin-Slack/actions/workflows/matomo-tests.yml/badge.svg?branch=5.x-dev)](https://github.com/matomo-org/plugin-Slack/actions/workflows/matomo-tests.yml)
```

Comment:
We probably don't want to specify the branch so that it uses the repo's default branch, which should be 5.x-dev right now.
```suggestion
[![Build Status](https://github.com/matomo-org/plugin-Slack/actions/workflows/matomo-tests.yml/badge.svg)](https://github.com/matomo-org/plugin-Slack/actions/workflows/matomo-tests.yml)
```